src/bindings.rs

```diff
@@ -400,6 +585,14 @@ fn make_cstring(value: impl Into<Vec<u8>>) -> Result<CString, ValueError> {
     CString::new(value).map_err(ValueError::StringWithZeroBytes)
 }
 
+/// Helper to construct null JsValue
+fn js_null_value() -> q::JSValue {
```

Comment:
I wanted to add this about 3 times already but was too lazy each time. :D 

Could probably use:
```suggestion
#[inline]
fn js_null_value() -> q::JSValue {
```

Reply:
But shouldn't compiler automatically inline it if the inlining is beneficial, as the function is used in the same file?